mlir-sycl/lib/Dialect/IR/SYCLOpsDialect.cpp

```diff
@@ -182,6 +182,10 @@ SYCLOpAsmInterface::getAlias(mlir::Type Type, llvm::raw_ostream &OS) const {
                << Ty.getDataType() << "_";
             return AliasResult::FinalAlias;
           })
+      .Case<mlir::sycl::StreamType>([&](auto Ty) {
+        OS << "sycl_" << decltype(Ty)::getMnemonic() << "_";
+        return AliasResult::OverridableAlias;
```

Comment:
Why is this `OverridableAlias`? Is there another way to write `sycl::stream`?

Reply:
Missed that. I don't think so. Can address in a different PR.

Reply:
@arnamoy10 please add it in 
```
      .Case<mlir::sycl::AssertHappenedType, mlir::sycl::BFloat16Type,
            mlir::sycl::KernelHandlerType>([&](auto Ty) {
        OS << "sycl_" << decltype(Ty)::getMnemonic() << "_";
        return AliasResult::FinalAlias;
      })
```